sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SonarRunner.java

```diff
@@ -32,8 +32,9 @@
  * Executes the sonar-runner script. In-process mode is not supported yet.
  *
  * @since 2.1
+ * @deprecated since 3.8 use {@link SonarScanner}
  */
-public final class SonarRunner extends Build<SonarRunner> {
+public class SonarRunner extends Build<SonarRunner> {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @Deprecated annotation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)